.circleci/config.yml

```diff
@@ -21,6 +21,10 @@ workflows:
                 debian_version: "stretch"
               - python_version: "3.5"
                 debian_version: "buster"
+      - quality:
```

Comment:
@MoisesGSalas what do you think about `quality-js` as job name to differentiate it from the python jobs?

Reply:
That would be a better name yes.